### ISSUE TITLE
Arbitrary Distribution of Individuals as Initial Conditions

### ISSUE THREAD

**Issue by Atreyu-94** [JuliaOcean/PlanktonIndividuals.jl#72]
Sorry, I haven't been able to implement the silicon cycle yet.
I had a question, is it possible that the initial conditions of individuals can be determined in an array as `temp` instead of a random distribution?

**Comment by Atreyu-94** [comment]
I was checking [here](https://github.com/JuliaOcean/PlanktonIndividuals.jl/blob/614223547c87a85eab030125ee8bf8e5f0d347fb/src/Plankton/CarbonMode/plankton_generation.jl#L30) but I didn't find that it could be done in the documentation either

**Comment by zhenwu0728** [comment]
Yes, it's doable. You can generate the individuals with a random distribution first. And after `PlanktonModel` is generated, you can change the attributes of individuals as you like, for example `model.individuals.phytos.sp1.data.Bm`.
